grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -996,15 +996,10 @@ class ProductAvailabilityService {
                 existingPA.quantityAllocated += remainingPA.quantityAllocated
                 existingPA.quantityNotPicked += remainingPA.quantityNotPicked
                 existingPA.quantityOnHold += remainingPA.quantityOnHold
-                existingPA. quantityAvailableToPromise += remainingPA.quantityAvailableToPromise
+                existingPA.quantityAvailableToPromise += remainingPA.quantityAvailableToPromise
```

Comment:
I know it's kind of long, but I'd prefer to spell existingPA out as existingProductAvailability for readability. Same with remainingPAs, et al. 

Remind me why we need to do this instead of just rerunning the product availability once all transactions have been "fixed".

Reply:
@jmiranda I did all the requested changes. I do not remember why I went this route. I only remember that I initially did with the refresh, but then I changed to this approach and pushed this version in the first commit/PR for this feature.